code/ATMOSPHERICS/atmospherics.dm

```diff
@@ -190,9 +191,14 @@ GLOBAL_DATUM_INIT(pipe_icon_manager, /datum/pipe_icon_manager, new())
 				"<span class='italics'>You hear ratchet.</span>")
 			investigate_log("was <span class='warning'>REMOVED</span> by [key_name(usr)]", "atmos")
 
+			for(var/obj/item/clothing/shoes/magboots/usermagboots in user.get_equipped_items())
+				if(usermagboots.gustprotection && usermagboots.magpulse)
+					safefromgusts = TRUE
+
 			//You unwrenched a pipe full of pressure? let's splat you into the wall silly.
 			if(unsafe_wrenching)
-				unsafe_pressure_release(user,internal_pressure)
+				if(safefromgusts) to_chat(user, "<span class='italics'>Your magboots cling to the floor as a great burst of wind bellows against you.")
+				else unsafe_pressure_release(user,internal_pressure)
```

Comment:
The preferred style for ifs is that you put the code that gets ran when it succeeds, on a new line, like this:
```suggestion
				if(safefromgusts) 
					to_chat(user, "<span class='italics'>Your magboots cling to the floor as a great burst of wind bellows against you.</span>")
				else 
					unsafe_pressure_release(user,internal_pressure)
```
Also, you are missing a `</span>` at the end of your `to_chat` (included in the suggestion).

Reply:
I only recently learned why I shouldn't code like this. thank you for the suggestion, I will commit this.